employees/views.py

```diff
@@ -451,10 +490,45 @@ def get_queryset(self) -> QuerySet:
         )
 
     def get_context_data(self, **kwargs: Any) -> dict:
+        def get_expected_hours_sum_for_all_projects(projects: QuerySet) -> Tuple[dict, datetime.timedelta]:
```

Comment:
If you're using same method more then one time it's good to export it to separate function and reuse it in some places. You can also create mixin class and inherit needed methods from this class

Reply:
Changed